setup.cfg

```diff
@@ -9,6 +9,13 @@ no-accept-encodings = True
 ignore = E731, E402, W504, C814
 max-line-length = 99
 
+[isort]
+line_length=99
+known_future_library=future
+multi_line_output=3
+default_section=FIRSTPARTY
```

Comment:
IMO using `known_first_party` instead and setting that to `raiden,raiden_contracts,raiden_libs` would be better (to avoid unknown libraries being treated as first party code).